lib/sepa_king/converter.rb

```diff
@@ -15,24 +15,19 @@ def convert(*attributes, options)
     end
 
     module InstanceMethods
-      def convert_text(value)
-        return unless value
-
-        value.to_s.
-          # Replace some special characters described as "Best practices" in Chapter 6.2 of this document:
-          # http://www.europeanpaymentscouncil.eu/index.cfm/knowledge-bank/epc-documents/sepa-requirements-for-an-extended-character-set-unicode-subset-best-practices/
-          gsub('€','E').
-          gsub('@','(at)').
-          gsub('_','-').
+      require 'csv'
 
-          # Replace linebreaks by spaces
-          gsub(/\n+/,' ').
+      EPC_MAPPING = CSV.read("lib/misc/epc_mapping.csv", col_sep: ",").
+        each_with_object(Hash.new('.')) { |v, mappings| mappings[v.first] = v.last }.
```

Comment:
Minor: Am I reading this Hash.new(".") right in that if a letter is not found, the `"."` will be offered instead? I guess that's a part of the Best Practices.